packages/appstore/src/components/create-wallet/create-wallet-provider.ts

```diff
@@ -0,0 +1,32 @@
+import { localize } from '@deriv/translations';
+
+const fiat_wallets = [
+    { light: 'IcAppstoreWalletAudLight', dark: 'IcAppstoreWalletAudLight' },
+    { light: 'IcAppstoreWalletEurLight', dark: 'IcAppstoreWalletEurLight' },
+    { light: 'IcAppstoreWalletGbpLight', dark: 'IcAppstoreWalletGbpLight' },
+    { light: 'IcAppstoreWalletUsdLight', dark: 'IcAppstoreWalletUsdLight' },
+];
+
+const crypto_wallets = [
+    { light: 'IcAppstoreWalletBitcoinLight', dark: 'IcAppstoreWalletBitcoinLight' },
+    { light: 'IcAppstoreWalletEthereumLight', dark: 'IcAppstoreWalletEthereumLight' },
+    { light: 'IcAppstoreWalletLitecoinLight', dark: 'IcAppstoreWalletLitecoinLight' },
+    { light: 'IcAppstoreWalletEusdtLight', dark: 'IcAppstoreWalletEusdtLight' },
+    { light: 'IcAppstoreWalletUsdtLight', dark: 'IcAppstoreWalletUsdtLight' },
+    { light: 'IcAppstoreWalletUsdcLight', dark: 'IcAppstoreWalletUsdcLight' },
+];
+
+const built_in_wallets = [
+    { light: 'IcAppstoreWalletP2pLight', dark: 'IcAppstoreWalletP2pLight' },
+    { light: 'IcAppstoreWalletPaLight', dark: 'IcAppstoreWalletPaLight' },
+];
+
+const wallets = [
+    { title: localize('Fiat currency wallets'), content: fiat_wallets },
```

Comment:
If you are using `localize` function, then we should change the title to `getTitle` function. This is because, when we change the language, it won't render the new language. We did the same thing for `title`s in `routes-config.js`.